*** ./mcop/buffer.cc.osr6	Wed Oct  5 06:09:52 2005
--- ./mcop/buffer.cc	Fri Jan 27 10:14:13 2006
***************
*** 101,109 ****
  void Buffer::writeString(const string& s) {
  	long len = s.size()+1;
  
! 	writeLong(len);
! 	contents.insert(contents.end(),reinterpret_cast<const unsigned char*>(s.c_str()),
! 		        reinterpret_cast<const unsigned char*>(s.c_str()+len));
  }
  
  void Buffer::writeStringSeq(const vector<string>& seq) {
--- 101,115 ----
  void Buffer::writeString(const string& s) {
  	long len = s.size()+1;
  
!         const unsigned char *s1, *e1;
!         unsigned char *s2, *e2;
! 
!         writeLong(len);
!         s1 = reinterpret_cast<const unsigned char*>(s.c_str());
!         e1 = reinterpret_cast<const unsigned char*>(s.c_str()+len);
!         s2 = const_cast<unsigned char *>(s1);
!         e2 = const_cast<unsigned char *>(e1);
!         contents.insert(contents.end(), s2, e2);
  }
  
  void Buffer::writeStringSeq(const vector<string>& seq) {
*** ./configure.osr6	Wed Oct  5 11:57:24 2005
--- ./configure	Fri Jan 27 10:14:13 2006
***************
*** 3178,3184 ****
  cat confdefs.h >>conftest.$ac_ext
  cat >>conftest.$ac_ext <<_ACEOF
  /* end confdefs.h.  */
- $ac_declaration
  #include <stdlib.h>
  int
  main ()
--- 3178,3183 ----
***************
*** 3224,3230 ****
  cat confdefs.h >>conftest.$ac_ext
  cat >>conftest.$ac_ext <<_ACEOF
  /* end confdefs.h.  */
- $ac_declaration
  int
  main ()
  {
--- 3223,3228 ----
***************
*** 3264,3274 ****
  rm -f conftest.err conftest.$ac_objext conftest.$ac_ext
  done
  rm -f conftest*
- if test -n "$ac_declaration"; then
-   echo '#ifdef __cplusplus' >>confdefs.h
-   echo $ac_declaration      >>confdefs.h
-   echo '#endif'             >>confdefs.h
- fi
  
  else
    echo "$as_me: failed program was:" >&5
--- 3262,3267 ----
***************
*** 3912,3918 ****
  cat confdefs.h >>conftest.$ac_ext
  cat >>conftest.$ac_ext <<_ACEOF
  /* end confdefs.h.  */
- $ac_declaration
  #include <stdlib.h>
  int
  main ()
--- 3905,3910 ----
***************
*** 3958,3964 ****
  cat confdefs.h >>conftest.$ac_ext
  cat >>conftest.$ac_ext <<_ACEOF
  /* end confdefs.h.  */
- $ac_declaration
  int
  main ()
  {
--- 3950,3955 ----
***************
*** 3998,4008 ****
  rm -f conftest.err conftest.$ac_objext conftest.$ac_ext
  done
  rm -f conftest*
- if test -n "$ac_declaration"; then
-   echo '#ifdef __cplusplus' >>confdefs.h
-   echo $ac_declaration      >>confdefs.h
-   echo '#endif'             >>confdefs.h
- fi
  
  ac_ext=c
  ac_cpp='$CPP $CPPFLAGS'
--- 3989,3994 ----
*** ./Configure-OSR6.osr6	Fri Jan 27 10:14:13 2006
--- ./Configure-OSR6	Fri Jan 27 10:14:37 2006
***************
*** 0 ****
--- 1,26 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ KDEDIR=/opt/kde350
+ QTDIR=/opt/kde350
+ export KDEDIR QTDIR
+ 
+ CFLAGS="-O2"
+ LDFLAGS="-z multidefs"
+ NM="/usr/ccs/bin/nm -p"
+ if [ "$1" = "-g" ]
+ then
+   CC="gcc -D_REENTRANT"
+   CXX="g++ -D_REENTRANT"
+   CFLAGS="$CFLAGS -Wall"
+ else
+   CC="cc -Kpthread -Kalloca"
+   CXX="CC -Kpthread -Kalloca"
+ fi
+ export CC CXX CFLAGS LDFLAGS NM
+ rm -f mout-configure
+ ./configure --prefix=$KDEDIR --with-qt-dir=$KDEDIR \
+             --x-includes=/usr/X11R6/include \
+             --x-libraries=/usr/X11R6/lib \
+             --host=$HOST --target=$HOST \
+             --build=$HOST 2>&1 | tee mout-configure
