*** ./source/defs.hh.osr6	Wed Oct 15 17:45:16 2003
--- ./source/defs.hh	Tue Jul  5 10:54:58 2005
***************
*** 8,21 ****
  #ifndef DEFS_HH
  #define DEFS_HH
  
! #define DEFAULT_DIR "../work/"
  
  /*
  FIXME:  remove asap, switching to file-configureable option
  */
  // path to html doc files
  #ifndef HTMLDIR
! #define HTMLDIR "../work/doc_html/"
  #endif
  
  
--- 8,21 ----
  #ifndef DEFS_HH
  #define DEFS_HH
  
! #define DEFAULT_DIR "/usr/share/avida"
  
  /*
  FIXME:  remove asap, switching to file-configureable option
  */
  // path to html doc files
  #ifndef HTMLDIR
! #define HTMLDIR "/usr/share/avida/doc_html/"
  #endif
  
  
*** ./Install-OSR6.osr6	Tue Jul  5 10:54:58 2005
--- ./Install-OSR6	Tue Jul  5 10:54:58 2005
***************
*** 0 ****
--- 1,11 ----
+ #!/bin/sh
+ 
+ MOUT=mout-install
+ D=/u/tmp/avida
+ DESTDIR=$D
+ export DESTDIR
+ 
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ make install DESTDIR=$D 2>&1 | tee $MOUT
*** ./Configure-OSR6.osr6	Tue Jul  5 10:54:58 2005
--- ./Configure-OSR6	Tue Jul  5 10:55:31 2005
***************
*** 0 ****
--- 1,14 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2"
+ NM="/usr/ccs/bin/nm -p"
+ M4=/usr/gnu/bin/m4
+ CC="gcc -D_REENTRANT"
+ CXX="g++ -D_REENTRANT"
+ CFLAGS="$CFLAGS -Wall"
+ export CC CXX CFLAGS NM M4
+ rm -f mout-configure
+ ./configure --enable-standard-prefix \
+             --bindir=/usr/bin --datadir=/usr/share/avida --prefix=/usr \
+             --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
