*** ./Install-OSR6.osr6	Tue Jul  5 15:30:41 2005
--- ./Install-OSR6	Tue Jul  5 15:30:41 2005
***************
*** 0 ****
--- 1,11 ----
+ #!/bin/sh
+ 
+ MOUT=mout-install
+ D=/u/tmp/bb
+ DESTDIR=$D
+ export DESTDIR
+ 
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ make install DESTDIR=$D 2>&1 | tee $MOUT
*** ./scene1.c.osr6	Thu Apr 26 08:52:55 2001
--- ./scene1.c	Tue Jul  5 15:31:48 2005
***************
*** 264,272 ****
  	"TEXT",
  	"MODE",
  	"",
! 	"DEVELOPED",
  	"UNDER",
! 	"LINUX",
  	"!",
  	"!",
  	"!",
--- 264,276 ----
  	"TEXT",
  	"MODE",
  	"",
! 	"DISPLAYED",
  	"UNDER",
! 	"SCO",
! 	"SCO",
! 	"SCO",
! 	"SCO",
! 	"SCO",
  	"!",
  	"!",
  	"!",
***************
*** 288,294 ****
--- 292,302 ----
  	1,
  	5,
  	3,
+ 	5,
+ 	4,
  	3,
+ 	2,
+ 	1,
  	1,
  	2,
  	3,
