*** ./configure.00	Thu May  4 09:50:26 2006
--- ./configure	Thu Apr 26 22:30:49 2007
***************
*** 10203,10209 ****
    lt_cv_deplibs_check_method=pass_all
    ;;
  
! sysv4 | sysv4.2uw2* | sysv4.3* | sysv5*)
    case $host_vendor in
    motorola)
      lt_cv_deplibs_check_method='file_magic ELF [0-9][0-9]*-bit [ML]SB (shared object|dynamic lib) M[0-9][0-9]* Version [0-9]'
--- 10203,10209 ----
    lt_cv_deplibs_check_method=pass_all
    ;;
  
! sysv4 | sysv4.2uw2* | sysv4.3*)
    case $host_vendor in
    motorola)
      lt_cv_deplibs_check_method='file_magic ELF [0-9][0-9]*-bit [ML]SB (shared object|dynamic lib) M[0-9][0-9]* Version [0-9]'
***************
*** 10227,10233 ****
    esac
    ;;
  
! sysv5OpenUNIX8* | sysv5UnixWare7* | sysv5uw[78]* | unixware7* | sysv4*uw2*)
    lt_cv_deplibs_check_method=pass_all
    ;;
  esac
--- 10227,10233 ----
    esac
    ;;
  
! sysv5* | unixware7* | sysv4*uw2*)
    lt_cv_deplibs_check_method=pass_all
    ;;
  esac
***************
*** 12204,12210 ****
        fi
        ;;
  
!     solaris* | sysv5*)
        if $LD -v 2>&1 | grep 'BFD 2\.8' > /dev/null; then
  	ld_shlibs=no
  	cat <<EOF 1>&2
--- 12204,12210 ----
        fi
        ;;
  
!     solaris*)
        if $LD -v 2>&1 | grep 'BFD 2\.8' > /dev/null; then
  	ld_shlibs=no
  	cat <<EOF 1>&2
***************
*** 12880,12908 ****
        runpath_var=LD_RUN_PATH
        ;;
  
!    sysv5OpenUNIX8* | sysv5UnixWare7* |  sysv5uw[78]* | unixware7*)
!       no_undefined_flag='${wl}-z ${wl}text'
        if test "$GCC" = yes; then
! 	archive_cmds='$CC -shared ${wl}-h ${wl}$soname -o $lib $libobjs $deplibs $compiler_flags'
        else
! 	archive_cmds='$CC -G ${wl}-h ${wl}$soname -o $lib $libobjs $deplibs $compiler_flags'
        fi
        runpath_var='LD_RUN_PATH'
        hardcode_shlibpath_var=no
        ;;
  
-     sysv5*)
-       no_undefined_flag=' -z text'
-       # $CC -shared without GNU ld will not create a library from C++
-       # object files and a static libstdc++, better avoid it by now
-       archive_cmds='$LD -G${allow_undefined_flag} -h $soname -o $lib $libobjs $deplibs $linker_flags'
-       archive_expsym_cmds='$echo "{ global:" > $lib.exp~cat $export_symbols | $SED -e "s/\(.*\)/\1;/" >> $lib.exp~$echo "local: *; };" >> $lib.exp~
-   		$LD -G${allow_undefined_flag} -M $lib.exp -h $soname -o $lib $libobjs $deplibs $linker_flags~$rm $lib.exp'
-       hardcode_libdir_flag_spec=
-       hardcode_shlibpath_var=no
-       runpath_var='LD_RUN_PATH'
-       ;;
- 
      uts4*)
        archive_cmds='$LD -G -h $soname -o $lib $libobjs $deplibs $linker_flags'
        hardcode_libdir_flag_spec='-L$libdir'
--- 12880,12898 ----
        runpath_var=LD_RUN_PATH
        ;;
  
!    sysv5* | unixware7*)
!       no_undefined_flag=''
        if test "$GCC" = yes; then
! 	archive_cmds='$CC -shared ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
! 	archive_expsym_cmds='$CC -shared ${wl}-Bexport:$export_symbols ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
        else
! 	archive_cmds='$CC -G ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
! 	archive_expsym_cmds='$CC -G ${wl}-Bexport:$export_symbols ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
        fi
        runpath_var='LD_RUN_PATH'
        hardcode_shlibpath_var=no
        ;;
  
      uts4*)
        archive_cmds='$LD -G -h $soname -o $lib $libobjs $deplibs $linker_flags'
        hardcode_libdir_flag_spec='-L$libdir'
***************
*** 16058,16066 ****
  	;;
      esac
      ;;
!   sysv5OpenUNIX8* | sysv5UnixWare7* | sysv5uw[78]* | unixware7*)
!     archive_cmds_need_lc_CXX=no
!     ;;
    tandem*)
      case $cc_basename in
        NCC)
--- 16048,16066 ----
  	;;
      esac
      ;;
!    sysv5* | unixware7*)
!       no_undefined_flag=''
!       if test "$GCC" = yes; then
! 	archive_cmds_CXX='$CC -shared ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
! 	archive_expsym_cmds_CXX='$CC -shared ${wl}-Bexport:$export_symbols ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
!       else
! 	archive_cmds_CXX='$CC -G ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
! 	archive_expsym_cmds_CXX='$CC -G ${wl}-Bexport:$export_symbols ${wl}-h ${wl}/usr/lib/$soname -o $lib $libobjs $deplibs $compiler_flags'
!       fi
!       runpath_var_CXX='LD_RUN_PATH'
!       hardcode_shlibpath_var_CXX=no
!       ;;
! 
    tandem*)
      case $cc_basename in
        NCC)
