*** ./Build-UW7.uw7	Tue Feb  5 13:29:18 2008
--- ./Build-UW7	Tue Feb  5 13:29:18 2008
***************
*** 0 ****
--- 1,33 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2 -I/usr/local/include"
+ LDFLAGS="-L/usr/local/lib"
+ INSTALL=/usr/gnu/bin/install
+ NM="nm -p"
+ M4=/usr/gnu/bin/m4
+ if [ "$1" = "-g" ]
+ then
+   CC="gcc -D_REENTRANT"
+   CXX="g++ -D_REENTRANT"
+   CFLAGS="$CFLAGS -Wall"
+ else
+   CC="cc -Kpthread -Kalloca"
+   CXX="CC -Kpthread -Kalloca"
+ fi
+ export CC CXX CFLAGS LDFLAGS NM M4 INSTALL
+ PCRE_CFLAGS="-I/usr/local/include"
+ PCRE_LIBS="-L/usr/local/lib -lpcre"
+ export PCRE_CFLAGS PCRE_LIBS
+ PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/lib/pkgconfig
+ export PKG_CONFIG_PATH
+ 
+ rm -f mout-configure
+ ./configure --prefix=/usr/local --sysconfdir=/etc --mandir=/usr/local/man \
+             --with-threads=posix --disable-shared --enable-static \
+             --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
+ 
+ MOUT=mout
+ [ "$1" ] && MOUT=mout-$1
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ gmake $* 2>&1 | tee $MOUT
*** ./Install-UW7.uw7	Tue Feb  5 13:29:18 2008
--- ./Install-UW7	Tue Feb  5 13:29:34 2008
***************
*** 0 ****
--- 1,12 ----
+ #!/bin/sh
+ 
+ MOUT=mout-install
+ D=/u/tmp/libxslt
+ DESTDIR=$D
+ export DESTDIR
+ 
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ gmake install DESTDIR=$D 2>&1 | tee $MOUT
+ 
