*** ./configure.00	Fri May  2 17:31:15 2003
--- ./configure	Sat Oct  1 16:50:59 2005
***************
*** 3297,3307 ****
    lt_cv_file_magic_test_file=/lib/libc.so
    ;;
  
! sysv5uw[78]* | sysv4*uw2*)
    lt_cv_deplibs_check_method=pass_all
    ;;
  
! sysv4 | sysv4.2uw2* | sysv4.3* | sysv5*)
    case $host_vendor in
    motorola)
      lt_cv_deplibs_check_method='file_magic ELF [0-9][0-9]*-bit [ML]SB (shared object|dynamic lib) M[0-9][0-9]* Version [0-9]'
--- 3297,3307 ----
    lt_cv_file_magic_test_file=/lib/libc.so
    ;;
  
! sysv5* | sysv4*uw2*)
    lt_cv_deplibs_check_method=pass_all
    ;;
  
! sysv4 | sysv4.2uw2* | sysv4.3*)
    case $host_vendor in
    motorola)
      lt_cv_deplibs_check_method='file_magic ELF [0-9][0-9]*-bit [ML]SB (shared object|dynamic lib) M[0-9][0-9]* Version [0-9]'
***************
*** 5272,5278 ****
      fi
      ;;
  
!   solaris* | sysv5*)
      if $LD -v 2>&1 | egrep 'BFD 2\.8' > /dev/null; then
        ld_shlibs=no
        cat <<EOF 1>&2
--- 5272,5278 ----
      fi
      ;;
  
!   solaris*)
      if $LD -v 2>&1 | egrep 'BFD 2\.8' > /dev/null; then
        ld_shlibs=no
        cat <<EOF 1>&2
***************
*** 5719,5730 ****
      ;;
  
    sysv5*)
!     no_undefined_flag=' -z text'
      # $CC -shared without GNU ld will not create a library from C++
      # object files and a static libstdc++, better avoid it by now
!     archive_cmds='$LD -G${allow_undefined_flag} -h $soname -o $lib $libobjs $deplibs $linker_flags'
!     archive_expsym_cmds='$echo "{ global:" > $lib.exp~cat $export_symbols | sed -e "s/\(.*\)/\1;/" >> $lib.exp~$echo "local: *; };" >> $lib.exp~
! 		$LD -G${allow_undefined_flag} -M $lib.exp -h $soname -o $lib $libobjs $deplibs $linker_flags~$rm $lib.exp'
      hardcode_libdir_flag_spec=
      hardcode_shlibpath_var=no
      runpath_var='LD_RUN_PATH'
--- 5719,5729 ----
      ;;
  
    sysv5*)
!     no_undefined_flag=''
      # $CC -shared without GNU ld will not create a library from C++
      # object files and a static libstdc++, better avoid it by now
!     archive_cmds='$LD -G${allow_undefined_flag} -h $rpath/$soname -o $lib $libobjs $deplibs $linker_flags'
!     archive_expsym_cmds='$LD -G${allow_undefined_flag} -Bexport:$export_symbols -h $rpath/$soname -o $lib $libobjs $deplibs $linker_flags'
      hardcode_libdir_flag_spec=
      hardcode_shlibpath_var=no
      runpath_var='LD_RUN_PATH'
*** ./Install-OSR6.00	Sat Oct  1 16:51:51 2005
--- ./Install-OSR6	Sat Oct  1 16:52:02 2005
***************
*** 0 ****
--- 1,12 ----
+ #!/bin/sh
+ 
+ MOUT=mout-install
+ D=/u/tmp/linc
+ DESTDIR=$D
+ export DESTDIR
+ 
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ make install DESTDIR=$D 2>&1 | tee $MOUT
+ 
