*** ./Configure-OSR6.osr6	Sat Sep  3 19:26:19 2005
--- ./Configure-OSR6	Sat Sep  3 19:27:31 2005
***************
*** 0 ****
--- 1,20 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2"
+ INSTALL=/usr/gnu/bin/install
+ NM=nm-p
+ M4=/usr/gnu/bin/m4
+ if [ "$1" = "-g" ]
+ then
+   CC="gcc -D_REENTRANT"
+   CXX="g++ -D_REENTRANT"
+   CFLAGS="$CFLAGS -Wall"
+ else
+   CC="cc -Kpthread -Kalloca"
+   CXX="CC -Kpthread -Kalloca"
+ fi
+ export CC CXX CFLAGS NM M4 INSTALL
+ rm -f mout-config mout-setup
+ ruby setup.rb config 2>&1 | tee mout-config
+ ruby setup.rb setup 2>&1 | tee mout-setup
*** ./Install-OSR6.osr6	Sat Sep  3 19:26:19 2005
--- ./Install-OSR6	Sat Sep  3 19:28:54 2005
***************
*** 0 ****
--- 1,6 ----
+ #!/bin/sh
+ 
+ MOUT=mout-install
+ 
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ ruby setup.rb install 2>&1 | tee $MOUT
