--- ./contrib/cube/Makefile.uw	2003-09-14 01:52:25.000000000 +0000
+++ ./contrib/cube/Makefile	2005-04-03 22:36:37.770000031 +0000
@@ -11,6 +11,8 @@
 DOCS = README.cube
 REGRESS = cube
 
+cube.o: cube.c
+	cc -Kthread -Kalloca -KPIC -I. -I../../src/include -I/usr/include  -c -o cube.o cube.c
 
 # cubescan is compiled as part of cubeparse
 cubeparse.o: cubescan.c
--- ./src/bin/pg_ctl/pg_ctl.sh.uw	2005-04-03 22:36:37.750000036 +0000
+++ ./src/bin/pg_ctl/pg_ctl.sh	2005-04-03 22:36:37.770000047 +0000
@@ -397,7 +397,7 @@
 	$silence_echo $ECHO_N "waiting for postmaster to start..."$ECHO_C
 	while :
 	do
-	    if ps -C postmaster >/dev/null 2>&1
+	    if /bin/ps -af | grep -v grep | grep -q postmaster >/dev/null 2>&1
 	    then
 		break;
 	    else
--- ./src/Makefile.shlib.uw	2003-10-20 01:34:33.000000000 +0000
+++ ./src/Makefile.shlib	2005-04-03 23:26:59.060000013 +0000
@@ -174,7 +174,7 @@
   else
     LINK.shared		= $(CC) -G
     endif
-  LINK.shared		+= -Wl,-z,text -Wl,-h,$(soname)
+  LINK.shared		+= -Wl,-z,text -Wl,-h,/usr/lib/postgresql/$(NAME)$(DLSUFFIX)
 endif
 
 ifeq ($(PORTNAME), svr4)
@@ -194,7 +194,7 @@
   else
     LINK.shared		= $(CC) -G
   endif
-  LINK.shared		+= -Wl,-z,text -Wl,-h,$(soname) 
+  LINK.shared		+= -Wl,-z,text -Wl,-h,/usr/lib/postgresql/$(NAME)$(DLSUFFIX) 
 endif
 
 ifeq ($(PORTNAME), cygwin)
@@ -238,7 +238,7 @@
 ifdef MK_NO_LORDER
 	$(LINK.static) $@ $^
 else
-	$(LINK.static) $@ `$(LORDER) $^ | tsort`
+	$(LINK.static) $@ `$(LORDER) $^ | /usr/ccs/bin/tsort`
 endif
 	$(RANLIB) $@
 
