*** ./Install-OSR6.osr6	Tue Mar 20 10:08:29 2007
--- ./Install-OSR6	Tue Mar 20 10:08:29 2007
***************
*** 0 ****
--- 1,7 ----
+ #!/bin/sh
+ 
+ D=/u/tmp/subversion
+ 
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ make install DESTDIR=$D 2>&1 | tee mout-install
*** ./Configure-OSR6.osr6	Tue Mar 20 10:08:29 2007
--- ./Configure-OSR6	Tue Oct  9 13:25:21 2007
***************
*** 0 ****
--- 1,24 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2"
+ NM="/usr/ccs/bin/nm -p"
+ if [ "$1" = "-g" ]
+ then
+   CC="gcc -D_REENTRANT"
+   CXX="g++ -D_REENTRANT"
+   CFLAGS="$CFLAGS -Wall"
+ else
+   CC="cc -Kpthread -Kalloca"
+   CXX="CC -Kpthread -Kalloca"
+ fi
+ export CC CXX CFLAGS NM
+ rm -f mout-configure
+ if [ -x configure ]
+ then
+     ./configure --prefix=/usr --with-jdk=/usr/java --without-berkeley-db \
+             --with-apr=/usr/bin/apr-1-config \
+             --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
+ else
+     [ -f Makefile.PL ] && perl Makefile.PL 2>&1 | tee mout-configure
+ fi
*** ./configure.osr6	Wed Jan 17 14:17:34 2007
--- ./configure	Tue Mar 20 10:08:29 2007
***************
*** 20629,20635 ****
  rm -f conftest.err conftest.$ac_objext \
        conftest$ac_exeext conftest.$ac_ext
  if test "$ac_cv_search_bindtextdomain" = no; then
!   for ac_lib in intl; do
      LIBS="-l$ac_lib  $ac_func_search_save_LIBS"
      cat >conftest.$ac_ext <<_ACEOF
  /* confdefs.h.  */
--- 20629,20635 ----
  rm -f conftest.err conftest.$ac_objext \
        conftest$ac_exeext conftest.$ac_ext
  if test "$ac_cv_search_bindtextdomain" = no; then
!   for ac_lib in gnuintl; do
      LIBS="-l$ac_lib  $ac_func_search_save_LIBS"
      cat >conftest.$ac_ext <<_ACEOF
  /* confdefs.h.  */
