*** ./Configure-OSR6.osr6	Sat Jun  3 21:06:10 2006
--- ./Configure-OSR6	Sat Jun  3 21:06:51 2006
***************
*** 0 ****
--- 1,26 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2"
+ AS="/usr/gnu/bin/gas"
+ NM="/usr/ccs/bin/nm -p"
+ 
+ # Taglib does partial template instantiation in a way only g++ understands
+ if [ "$1" = "-c" ]
+ then
+   CC="cc -Kpthread -Kalloca"
+   CXX="CC -Kpthread -Kalloca"
+ else
+   CC="gcc -D_REENTRANT"
+   CXX="g++ -D_REENTRANT"
+   CFLAGS="$CFLAGS -Wall"
+ fi
+ export AS CC CXX CFLAGS NM
+ rm -f mout-configure
+ if [ -x configure ]
+ then
+     ./configure --prefix=/usr --program-prefix="" \
+             --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
+ else
+     [ -f Makefile.PL ] && perl Makefile.PL 2>&1 | tee mout-configure
+ fi
*** ./Install-OSR6.osr6	Sat Jun  3 21:06:10 2006
--- ./Install-OSR6	Sat Jun  3 21:06:10 2006
***************
*** 0 ****
--- 1,7 ----
+ #!/bin/sh
+ 
+ D=/u/tmp/taglib-1.4
+ 
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ make install DESTDIR=$D 2>&1 | tee mout-install
