*** ./src/wget.h.osr6 Tue Oct  7 17:05:45 2003
--- ./src/wget.h Sun May 29 19:07:59 2005
***************
*** 97,106 ****
  # define DEBUGP(x) DO_NOTHING
  #endif /* not ENABLE_DEBUG */
  
- /* Make gcc check for the format of logmsg() and debug_logmsg().  */
- #ifdef __GNUC__
- # define GCC_FORMAT_ATTR(a, b) __attribute__ ((format (printf, a, b)))
- #else  /* not __GNUC__ */
  # define GCC_FORMAT_ATTR(a, b)
  #endif /* not __GNUC__ */
  
--- 97,102 ----
  # define DEBUGP(x) DO_NOTHING
  #endif /* not ENABLE_DEBUG */
  
  # define GCC_FORMAT_ATTR(a, b)
  
  /* These are from log.c, but they are used everywhere, so we declare
***************
*** 102,108 ****
  # define GCC_FORMAT_ATTR(a, b) __attribute__ ((format (printf, a, b)))
  #else  /* not __GNUC__ */
  # define GCC_FORMAT_ATTR(a, b)
- #endif /* not __GNUC__ */
  
  /* These are from log.c, but they are used everywhere, so we declare
     them here.  */
--- 98,103 ----
  #endif /* not ENABLE_DEBUG */
  
  # define GCC_FORMAT_ATTR(a, b)
  
  /* These are from log.c, but they are used everywhere, so we declare
     them here.  */
*** ./src/connect.c.osr6 Sat Nov  1 06:08:22 2003
--- ./src/connect.c Sun May 29 19:07:59 2005
***************
*** 327,334 ****
    DEBUGP (("Master socket fd %d bound.\n", msock));
    if (!*port)
      {
!       socklen_t sa_len = sockaddr_len ();
!       if (getsockname (msock, &srv.sa, &sa_len) < 0)
  	{
  	  CLOSE (msock);
  	  msock = -1;
--- 327,334 ----
    DEBUGP (("Master socket fd %d bound.\n", msock));
    if (!*port)
      {
!       socklen_t my_sa_len = sockaddr_len ();
!       if (getsockname (msock, &srv.sa, &my_sa_len) < 0)
  	{
  	  CLOSE (msock);
  	  msock = -1;
*** ./src/host.c.osr6 Sat Oct 25 18:38:25 2003
--- ./src/host.c Sun May 29 19:07:59 2005
***************
*** 53,59 ****
  #  include <arpa/inet.h>
  # endif
  # include <netdb.h>
! # define SET_H_ERRNO(err) ((void)(h_errno = (err)))
  #endif /* WINDOWS */
  
  #ifndef NO_ADDRESS
--- 53,59 ----
  #  include <arpa/inet.h>
  # endif
  # include <netdb.h>
! # define SET_H_ERRNO(err) ((void)(errno = (err)))
  #endif /* WINDOWS */
  
  #ifndef NO_ADDRESS
*** ./Configure-OSR6.osr6 Sun May 29 19:08:00 2005
--- ./Configure-OSR6 Sun May 29 19:08:00 2005
***************
*** 0 ****
--- 1,10 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2"
+ CC="cc -Kpthread -Kalloca"
+ CXX="CC -Kpthread -Kalloca"
+ export CC CXX CFLAGS
+ rm -f mout-configure
+ ./configure --prefix=/usr/gnu \
+             --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
*** ./configure.osr6 Tue Nov 11 15:42:20 2003
--- ./configure Sun May 29 19:09:10 2005
***************
*** 13839,13846 ****
  
  
  
!       echo "$as_me:$LINENO: checking for gettext in -lintl" >&5
! echo $ECHO_N "checking for gettext in -lintl... $ECHO_C" >&6
  if test "${ac_cv_lib_intl_gettext+set}" = set; then
    echo $ECHO_N "(cached) $ECHO_C" >&6
  else
--- 13839,13846 ----
  
  
  
!       echo "$as_me:$LINENO: checking for gettext in -lgnuintl" >&5
! echo $ECHO_N "checking for gettext in -lgnuintl... $ECHO_C" >&6
  if test "${ac_cv_lib_intl_gettext+set}" = set; then
    echo $ECHO_N "(cached) $ECHO_C" >&6
  else
***************
*** 13845,13851 ****
    echo $ECHO_N "(cached) $ECHO_C" >&6
  else
    ac_check_lib_save_LIBS=$LIBS
! LIBS="-lintl  $LIBS"
  cat >conftest.$ac_ext <<_ACEOF
  #line $LINENO "configure"
  /* confdefs.h.  */
--- 13845,13851 ----
    echo $ECHO_N "(cached) $ECHO_C" >&6
  else
    ac_check_lib_save_LIBS=$LIBS
! LIBS="-lgnuintl  $LIBS"
  cat >conftest.$ac_ext <<_ACEOF
  #line $LINENO "configure"
  /* confdefs.h.  */
***************
*** 13895,13901 ****
  echo "${ECHO_T}$ac_cv_lib_intl_gettext" >&6
  if test $ac_cv_lib_intl_gettext = yes; then
  
!                 LIBS="-lintl $LIBS"
  	cat >>confdefs.h <<\_ACEOF
  #define HAVE_GETTEXT 1
  _ACEOF
--- 13895,13901 ----
  echo "${ECHO_T}$ac_cv_lib_intl_gettext" >&6
  if test $ac_cv_lib_intl_gettext = yes; then
  
!                 LIBS="-lgnuintl $LIBS"
  	cat >>confdefs.h <<\_ACEOF
  #define HAVE_GETTEXT 1
  _ACEOF
