--- include/config.h.00	2004-09-28 21:56:58.000000000 +0000
+++ include/config.h	2004-09-28 22:10:08.610000006 +0000
@@ -859,10 +859,10 @@
 #define PACKAGE_VERSION "20040914"
 
 /* Define to the soname of the libcrypto library. */
-#define SONAME_LIBCRYPTO "libcrypto.so.0
+#define SONAME_LIBCRYPTO "libcrypto.so.0"
 
 /* Define to the soname of the libcups library. */
-#define SONAME_LIBCUPS "libcups.so.2
+#define SONAME_LIBCUPS "libcups.so.2"
 
 /* Define to the soname of the libcurses library. */
 #define SONAME_LIBCURSES "libcurses.so"
@@ -889,7 +889,7 @@
 #define SONAME_LIBNCURSES "libncurses.so"
 
 /* Define to the soname of the libssl library. */
-#define SONAME_LIBSSL "libssl.so.0
+#define SONAME_LIBSSL "libssl.so.0"
 
 /* Define to the soname of the libtxc_dxtn library. */
 #define SONAME_LIBTXC_DXTN "libtxc_dxtn.so"
