--- ./Configure-OSR6.00	2005-06-12 22:09:00.000000000 -0700
+++ ./Configure-OSR6	2005-06-12 22:04:51.663241009 -0700
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+HOST=i586-sco-sysv5
+CFLAGS="-O2"
+NM="/usr/ccs/bin/nm -p"
+CC="cc -Kpthread -Kalloca"
+CXX="CC -Kpthread -Kalloca"
+export CC CXX CFLAGS NM
+rm -f mout-configure
+./configure --prefix=/usr --disable-ptys --disable-sigint \
+            --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
