*** ./Configure-UW7.uw7	Sun Mar 18 12:35:00 2007
--- ./Configure-UW7	Sun Mar 18 12:37:09 2007
***************
*** 0 ****
--- 1,26 ----
+ #!/bin/sh
+ #
+ HOST=i586-sco-sysv5
+ CFLAGS="-O2"
+ INSTALL=/usr/gnu/bin/install
+ NM="nm -p"
+ M4=/usr/gnu/bin/m4
+ GIMP_TOOL=/usr/bin/gimptool-2.0
+ GIMP_CONFIG=/usr/bin/gimptool-2.0
+ if [ "$1" = "-g" ]
+ then
+   CC="gcc -D_REENTRANT"
+   CXX="g++ -D_REENTRANT"
+   CFLAGS="$CFLAGS -Wall"
+ else
+   CC="cc -Kpthread -Kalloca"
+   CXX="CC -Kpthread -Kalloca"
+ fi
+ export CC CXX CFLAGS NM M4 INSTALL GIMP_CONFIG GIMP_TOOL
+ ac_cv_type_u_int8_t=yes
+ ac_cv_type_u_int16_t=yes
+ ac_cv_type_u_int32_t=yes
+ export ac_cv_type_u_int8_t ac_cv_type_u_int16_t ac_cv_type_u_int32_t
+ rm -f mout-configure
+ ./configure --prefix=/usr --sysconfdir=/etc --enable-gimp --disable-nls \
+             --host=$HOST --target=$HOST --build=$HOST 2>&1 | tee mout-configure
*** ./Install-UW7.uw7	Sun Mar 18 12:35:00 2007
--- ./Install-UW7	Sun Mar 18 12:35:00 2007
***************
*** 0 ****
--- 1,12 ----
+ #!/bin/sh
+ 
+ MOUT=mout-install
+ D=/u/tmp/xsane
+ DESTDIR=$D
+ export DESTDIR
+ 
+ [ -f $MOUT ] && mv $MOUT $MOUT-prev
+ [ -d $D ] && rm -rf $D
+ mkdir -p $D
+ make install DESTDIR=$D 2>&1 | tee $MOUT
+ 
